src/olympia/migrations/965-add-user-is-public-and-drop-is-verified.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE users
+    ADD COLUMN `public` boolean DEFAULT false,
+    DROP COLUMN `is_verified`;
+
+/* AUTHOR_ROLE_DEV = 4
+AUTHOR_ROLE_OWNER = 5
+STATUS_PUBLIC = 4 */
+
+UPDATE users, addons_users, addons SET users.`public`=true
+    WHERE users.`id`=addons_users.`user_id` and
+          addons_users.`role` in (4, 5) and
+          addons_users.`listed`=true and
+          addons_users.`addon_id`=addons.`id` and
+          addons.`status`=4;
```

Comment:
`in` and `and` should be uppercase and please add whitespaces before and after `=`